pyop/pyfunc.cc

```diff
@@ -399,7 +399,7 @@ void PyCustomOpDef::AddOp(const PyCustomOpDef* cod) {
 
   // No need to protect against concurrent access, GIL is doing that.
   auto val = std::make_pair(op_domain, std::vector<PyCustomOpFactory>());
-  const auto [it_domain_op, success] = PyOp_container().insert(val);
+  auto [it_domain_op, success] = PyOp_container().insert(val);
   assert(success || !it_domain_op->second.empty());
   it_domain_op->second.emplace_back(PyCustomOpFactory(cod, op_domain, op));
```

Comment:
At line 404 you probably do not need to have the "PyCustomOpFactory" string.

```
  it_domain_op->second.emplace_back(cod, op_domain, op);
```

Reply:
good to know, thanks.